blockchain/v0/reactor.go

```diff
@@ -298,6 +300,9 @@ func (r *Reactor) processBlockchainCh() {
 		case <-r.closeCh:
 			r.Logger.Debug("stopped listening on blockchain channel; closing...")
 			return
+
+		case envelop := <-r.peerUpdatesCh:
+			r.blockchainCh.Out <- envelop
```

Comment:
nit: can we put this case above the close case please?

Reply:
Done